Rakefile

```diff
@@ -4,9 +4,7 @@ require "rubocop/rake_task"
 
 ENV["RAILS_ENV"] ||= "test"
 
-RSpec::Core::RakeTask.new(:spec).tap do |rspec_task|
-  rspec_task.rspec_opts = "--fail-fast"
-end
+RSpec::Core::RakeTask.new(:spec)
```

Comment:
It's a bit confusing when making a change, see a different test fail (because of the randomised order and fail-fast), but you basically didn't fix anything.